src/inject/fallback.ts

```diff
@@ -1,8 +1,8 @@
-const fallbackStyle = document.createElement('style');
-fallbackStyle.textContent = 'html,body,body :not(iframe) { background-color:#181a1b!important;border-color:#776e62!important;color:#e8e6e3!important;';
-document.documentElement.appendChild(fallbackStyle);
-// After inserting into DOM no need for any performance reducing so all `meta/optional` things are being handled.
-// Also using document.documentElement so it's no need to wait until the `head` element is loaded.
-fallbackStyle.media = 'screen';
-fallbackStyle.classList.add('darkreader');
-fallbackStyle.classList.add('darkreader--fallback');
+if (matchMedia('(prefers-color-scheme: dark)').matches) {
```

Comment:
long time wanted feature again

Reply:
but I think you have forgot a `!` before `matchMedia()`.

Reply:
This isn't a feature, but rather a check to see if it should block white flash or not.

Reply:
`prefers-color-scheme: dark` check is incorrect, though. You should check the darkreader's theme preference to choose whether to apply whiteflash prevention instead.

Reply:
prefers-color-scheme: dark check is incorrect, though. You should check the darkreader's theme preference to choose whether to apply whiteflash prevention instead.

Reply:
Okay see, that is the problem I have `16.66....ms` to get this code executed and evaluated etc. Now going to the chrome.storage API would first need to get local API(Fast enough), but than also to check if sync is enabled so than It will go to sync server and get the data. Well I can tell, that isn't fast enough the be executed within 1 Render frame. So going trough the initial start procedure Isn't going to work as with this PR I'm bypassing it for this exact reason.

Reply:
And the check is correct. It was suggested by the owner of Dark Reader @alexanderby https://github.com/darkreader/darkreader/pull/3533#discussion_r479426454